apartments/tests.py

```diff
@@ -1,85 +1,47 @@
-from users.models import User
 from apartments.models import Apartment, City
 from .forms import ApartmentCreationForm
 from django.urls import reverse
 import pytest
 
 
-@pytest.fixture
-def new_user():
-    user = User.objects.create_user(
-        email='test@test.com',
-        first_name='first_name',
-        last_name='last_name',
-        birth_date='1900-01-01',
-        password='password'
-    )
-    user.save()
-    return user
-
-
-@pytest.fixture
-def new_city():
-    city = City(cityName='some_city')
-    city.save()
-    return city
-
-
-@pytest.fixture
-def new_apartment(new_user, new_city):
-    apartment = Apartment(
-        owner=new_user,
-        city=new_city,
-        address='street',
-        rent=4500,
-        num_of_roomates=2,
-        num_of_rooms=3,
-        start_date='2021-1-1',
-        about='Hey!',
-        image_url='www.some-url.com',
-    )
-    apartment.save()
-    return apartment
-
-
 @pytest.mark.django_db
 class TestModels:
 
-    def test_city_creation(self, new_city):
-        assert new_city is not None
-        assert new_city.cityName == 'some_city'
+    def test_city_creation(self, city_model):
+        assert city_model is not None
+        assert city_model.cityName == 'nice_city'
 
-    def test_get_city(self, new_city):
-        check_city = City.objects.get(cityName=new_city.cityName)
-        assert check_city == new_city
+    def test_get_city(self, city_model):
+        check_city = City.objects.get(cityName=city_model.cityName)
+        assert check_city == city_model
 
-    def test_apartment_creation(self, new_apartment):
-        assert new_apartment is not None
-        assert new_apartment.address == 'street'
-        assert new_apartment.rent == 4500
-        assert new_apartment.is_relevant is True
+    def test_apartment_creation(self, apartment_model):
+        assert apartment_model is not None
+        assert apartment_model.address == 'street'
+        assert apartment_model.rent == 4500
+        assert apartment_model.is_relevant is True
```

Comment:
as above, no need to assert apartment_model is not none as it is implicit.
no need to use `is True`, `assert apartment_model.is_relevant` will do the same thing